custom/icds_reports/reports/take_home_ration.py

```diff
@@ -36,6 +36,8 @@ def _format_infrastructure_data(data):
 
         query_set = TakeHomeRationMonthly.objects.filter(**filters).order_by(*order_by)
 
+        filters = [['Generated at', india_now()]]
```

Comment:
Why is this changed? @challabeehyv

Reply:
This looks changed by mistake?

Reply:
it is intentional. Actually there is a difference in india_now from test case to actual implementation by 1 sec. The time is getting assigned post query which is inducing some delay. So moved it a but higher to eliminate the lag